plug-ins/sample-data-import.ny

```diff
@@ -40,11 +40,11 @@ $control bad-data (_ "Invalid data handling") choice (("ThrowError" (_ "Throw Er
         t)))
 
 ;; ':new' creates a new class 'streamreader'
-;; 'filestream' and 'chanel' are its instance variables.
+;; 'filestream' and 'channel' are its instance variables.
 ;; (every object of class 'streamreader' has its own
 ;; copy of these variables)
 (setq streamreader
-  (send class :new '(filestream chanel)))
+  (send class :new '(filestream channel)))
```

Comment:
Is this OK?

Reply:
LGTM - @SteveDaulton confirm?

Reply:
> LGTM - @SteveDaulton confirm?

I can tell you that the fix is incomplete and incorrect, unless you change all "chanel" to "channel" and there is in fact more than one occurrence.

Reply:
@Paul-Licameli JFTR, i did a sourcecode-wide search for 'chanel' and came up with nothing. What am I missing?

Reply:
You are right, I was confused

Reply:
Looks OK to me.